### PR TITLE
test/factories: move poll factories here

### PR DESCRIPTION
**Pull request by Rineee**

### CODE REVIEW

**Review by fuzzylogic2000 — COMMENTED** [review]
We should also delete the ones that are in the tests folder and then also use these in a4. (Don't mind me if that was the plan anyway, just going through the PRs.)

**Comment by Rineee** [comment]
> We should also delete the ones that are in the tests folder and then also use these in a4. (Don't mind me if that was the plan anyway, just going through the PRs.)

Haha, yes that was the plan anyway, just made this PR for testing it in mB, but its ready now!

**Comment by Rineee** [comment]
But now everything is gone? Dont get it..

**Comment by fuzzylogic2000** [comment]
@Rineee This is weird! But when looking into the code of the branch, the changes are there! So, I guess we merge and deal with it, if that doesn't go as expected?!

**Comment by Rineee** [comment]
> @Rineee This is weird! But when looking into the code of the branch, the changes are there! So, I guess we merge and deal with it, if that doesn't go as expected?!

Yes, lets do that and see what happens!

File: adhocracy4/test/factories/polls.py [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+
 
 class PollFactory(factory.django.DjangoModelFactory):
 
     class Meta:
         model = models.Poll
 
-    creator = factory.SubFactory(UserFactory)
-    module = factory.SubFactory(factories.ModuleFactory)
+    creator = factory.SubFactory(USER_FACTORY)
```
Comment (fuzzylogic2000) [inline adhocracy4/test/factories/polls.py:16]: Right! And now the code changes are also back! Just merge whenever it's ready!

**Pull request merged**